tests/phpunit/HeaderConflictsTest.php

```diff
@@ -10,9 +12,10 @@
 
 class HeaderConflictsTest extends TestCase
 {
-    public function testObjectArrayCombinedConflictObject()
+    public function testObjectArrayCombinedConflictObject(): void
```

Comment:
Testy som presunul do adresara `phpunit`. 

V testoch je par kozmetickych zmien, ktore funguju aj v tejto starej verzii balickov, a su pripravou pre novu verziu. 
Nechcelo sa mi to uz zbytocne oddelovat do osobitneho PR, snad to nevadi.
Ak je to v tomto PR, kde sa nemeni `drc` kod, tak je aspon jasne, ze funkcnost zostava nezmenena.